build-setup.sh

```diff
@@ -88,7 +88,15 @@ fi
 $RDIR/scripts/init-submodules-no-riscv-tools.sh $FORCE_FLAG
 
 if [ "$SKIP_TOOLCHAIN" = false ]; then
-    $RDIR/scripts/build-toolchain-extra.sh $FORCE_FLAG $TOOLCHAIN
+    if [ "$SKIP_CONDA" = false ]; then
+        PREFIX=$CONDA_PREFIX/$TOOLCHAIN
+    else
+        if [ -z "$RISCV" ] ; then
+            error "ERROR: If --skip-conda used, \$RISCV variable must be defined."
+        fi
+        PREFIX=$RISCV
+    fi
+    $RDIR/scripts/build-toolchain-extra.sh $TOOLCHAIN -p $PREFIX
```

Comment:
What is the use case for `skip-toolchain-extra`? It might be cleaner to remove this option entirely? Building the toolchain-extra components should be very fast.

Reply:
No good reason that I could think of. In the past, I tried to make it so that the `build-setup.sh` script could opt out of running particular scripts but I agree with you that most people don't mind the extra runtime (and if they do, then they are most likely power users who would like to run the scripts separately).